docs/executor_swagger.md

```diff
@@ -81,22 +81,10 @@ ownership management and SELinux relabeling.
 
 | Name | Type | Go type | Required | Default | Description | Example |
 |------|------|---------|:--------:| ------- |-------------|---------|
-| fsType | string| `string` |  | | fsType is the filesystem type of the volume that you want to mount.
-Tip: Ensure that the filesystem type is supported by the host operating system.
-Examples: "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.
-More info: https://kubernetes.io/docs/concepts/storage/volumes#awselasticblockstore
-TODO: how do we prevent errors in the filesystem from compromising the machine
-+optional |  |
-| partition | int32 (formatted integer)| `int32` |  | | partition is the partition in the volume that you want to mount.
-If omitted, the default is to mount by volume name.
-Examples: For volume /dev/sda1, you specify the partition as "1".
-Similarly, the volume partition for /dev/sda is "0" (or you can leave the property empty).
-+optional |  |
-| readOnly | boolean| `bool` |  | | readOnly value true will force the readOnly setting in VolumeMounts.
-More info: https://kubernetes.io/docs/concepts/storage/volumes#awselasticblockstore
-+optional |  |
-| volumeID | string| `string` |  | | volumeID is unique ID of the persistent disk resource in AWS (Amazon EBS volume).
-More info: https://kubernetes.io/docs/concepts/storage/volumes#awselasticblockstore |  |
+| fsType | string| `string` |  | | fsType is the filesystem type of the volume that you want to mount.</br>Tip: Ensure that the filesystem type is supported by the host operating system.</br>Examples: "ext4", "xfs", "ntfs". Implicitly inferred to be "ext4" if unspecified.</br>More info: https://kubernetes.io/docs/concepts/storage/volumes#awselasticblockstore</br>TODO: how do we prevent errors in the filesystem from compromising the machine</br>+optional |  |
```

Comment:
Looks like most of the changes in this file are newline -> HTML line break `<br>`.

Or, in other words, this PR/the `go-swagger` update specifically, fixes #7615